css/theme-red.css

```diff
@@ -338,6 +338,52 @@ h6,
   line-height: 24px;
   font-weight: 700;
 }
+.text-extend{
+  position: absolute;
+  bottom: 0px;
+  left:30%:;
+  }
+
+@media only screen and (max-width: 990px) {
+    .text-extend {
+  position:absolute;
+  bottom:0px; 
+  top: 100%;
+  left: 20%;
+  }
+}
+
+@media only screen and (max-width: 770px) {
+    .text-extend {
+  position:absolute;
+  top: 50%;
+  left: 40%;
+  }
+}
+
+@media only screen and (max-width: 413px) {
+    .text-extend {
+  position:absolute;
+  top: 60%;
+  left: 34%;
+  }
+}
+
+@media only screen and (max-width: 360px) {
+    .text-extend {
+  position:absolute;
+  top: 70%;
+  left: 40%;
+  }
+}
+
+@media only screen and (max-width: 320px) {
```

Comment:
instead of modifying the theme can't you use bootstrap? It will make the code more stable. Currently your design is not working out of the box!

Reply:
Through bootstrap only some breakpoints could be accessed thats why @abishekvashok